src/promptflow-devkit/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -210,7 +211,7 @@ def get_pfs_version():
 def is_pfs_service_healthy(pfs_port) -> bool:
     """Check if pfs service is running and pfs version matches pf version."""
     try:
-        response = requests.get("http://localhost:{}/heartbeat".format(pfs_port))
+        response = requests.get(f"http://{PF_SERVICE_HOST}:{pfs_port}/heartbeat")
```

Comment:
one silly question, may update the answer to the PR description: what will this change bring to us? perf? or something else?

Reply:
(just curious, feel free to resolve this

Reply:
perf improve because DNS delay, detailed info is looged in https://msdata.visualstudio.com/Vienna/_workitems/edit/3146687/?view=edit